colcon_bundle/verb/assets/workspace_setup.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+COLCON_CURRENT_PREFIX=$BUNDLE_CURRENT_PREFIX . $BUNDLE_CURRENT_PREFIX/workspace/setup.sh
```

Comment:
Could we have setup.sh in the top level directory so that all the overlays are consistent in their directory structure.

Reply:
Yep, I fixed that.